routemaster/models/subscriber.rb

```diff
@@ -79,6 +79,28 @@ def uuid
       _attributes['uuid']
     end
 
+    def health_points
+      _attributes.fetch('health_points', '100').to_i
+    end
+
+    def change_health_by(offset)
+      new_value = _redis_lua_run(
+        'subscriber_change_health_by',
+        keys: [_key],
+        argv: [offset]
+      )
+      @_attributes = nil
+      new_value
+    end
+
+    def last_attempted_at
+      _attributes['last_attempted_at']&.to_i
```

Comment:
Ah, I get what you're saying then. So if it can return `nil`, won't some code fail elsewhere at the first attempt?  E.g. `Throttle#_stale_enough?` will fail if the timestamp is `nil`.